R/derive_adeg_params.R

```diff
@@ -6,10 +6,9 @@
 #'
 #' **Note:** This is a wrapper function for the more generic `derive_param_computed()`.
 #'
-#' @param dataset Input dataset
-#'
-#'   The variables specified by the `by_vars` and the `unit_var` parameter,
-#'   `PARAMCD`, and `AVAL` are expected.
+#' @param dataset
+#'   `r roxygen_param_dataset(expected_vars = c("by_vars", "get_unit_expr"))`
+#'   `PARAMCD`, and `AVAL` are expected as well.
```

Comment:
Other examples of explicit column names from `derive_var_atoxgr()`: 

Whats best way to handle?
![image](https://github.com/pharmaverse/admiral/assets/43018762/4a741f3f-bb38-4c77-9ec1-c543c7450917)

Reply:
@pharmaverse/admiral Do you all have thoughts on how to handle phrasing of explicitly named variables vs the ones we plug into function arguments?